modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/util/ConfigurationModelNameComparator.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.configuration.admin.web.util;
+
+import com.liferay.configuration.admin.web.model.ConfigurationModel;
+import com.liferay.portal.kernel.language.LanguageUtil;
+import com.liferay.portal.kernel.util.ResourceBundleLoader;
+
+import java.util.Comparator;
+import java.util.ResourceBundle;
+
+/**
+ * @author Andrew Betts
+ */
+public class ConfigurationModelNameComparator
```

Comment:
It looks like the main difference between this class and the existing ConfigurationModelComparator is that this one takes localization into account. If so the name should reflect that. So the name should be LocalizedConfigurationModelComparator, ConfigurationModelLocalizedComprator or something like that.